pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiQuestionValidationCached.java

```diff
@@ -57,15 +57,15 @@ public long insert(long questionId, long validationId) {
         } else {
             List<RuleDto> validations = null;
             try {
-                validations = questionIdToValidationsCache.get(questionDto.getId());
+                validations = questionIdToValidationsCache.get(questionDto.getActivityId());
```

Comment:
Hmmm, I'm getting a bit confused -- the variable name is `questionIdToValidationsCache` but we're looking up by activity id?

Reply:
yes, agree. It needs to be renamed.